pkgs/top-level/python-packages.nix

```diff
@@ -16728,6 +16728,27 @@ in modules // {
   };
 
 
+  shortuuid = buildPythonPackage rec {
+    name = "shortuuid-${version}";
+    version = "0.4.2";
+
+    disabled = isPy26;
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/shortuuid/${name}.tar.gz";
+      md5 = "142e3ae4e7cd32d41a71deb359db4cfe";
```

Comment:
Generally yes. The md5 is the one which is published on pypi, making it trivial to add. So if it's not a huge problem, I'd stick with that.